proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -233,6 +265,14 @@ message EnumValue {
   // Paragraph newlines (double new lines) are respected, however single newlines are not.
   // Note that any leading and trailing `//` or spaces within a `/* */` block will be stripped.
   string description = 3;
+  EnumValueOptions enum_value_options = 4;
+}
+
+// EnumValueOptions provide information for the documentation of options for an enum value.
+message EnumValueOptions {
+  reserved "UninterpretedOptions";
```

Comment:
Is there a reason why this particular version of `uninterpreted_options` is in camel case?

Reply:
Nope, certainly not